evm_loader/program/src/account_storage/backend.rs

```diff
@@ -151,14 +152,12 @@ impl<'a> AccountStorage for ProgramAccountStorage<'a> {
         OwnedAccountInfo::from_account_info(self.program_id, info)
     }
 
-    fn clone_solana_account_partial(
-        &self,
-        address: &Pubkey,
-        offset: usize,
-        len: usize,
-    ) -> Option<OwnedAccountInfoPartial> {
+    fn map_solana_account<F, R>(&self, address: &Pubkey, action: F) -> R
+    where
+        F: FnOnce(&AccountInfo) -> R,
+    {
         let info = self.solana_accounts[address];
```

Comment:
NeonEVM fails on missing address in solana_accounts. Is it ok?

Reply:
Yes. In this case it need to fail